rust_snuba/src/processors/generic_metrics.rs

```diff
@@ -339,6 +339,16 @@ fn should_use_killswitch(config: Result<Option<String>, Error>, use_case: &Messa
     false
 }
 
+fn parse_materialization_version(config: Result<Option<String>, Error>) -> u8 {
+    if let Some(mat_version) = config.ok().flatten() {
+        let mat_version_int = mat_version.parse::<u8>().unwrap();
+        return mat_version_int;
+    }
+
+    // return the currently used materialization version if runtime config is not set
+    2u8
```

Comment:
I have changed the function so that it only applies to distributions. That was my original intention, does this refactor work/make sense? Since we are only changing distributions matview I figured that it makes sense we have a function and runtime config for distributions only.